notion/store.py

```diff
@@ -289,6 +289,7 @@ def call_load_page_chunk(self, page_id):
 
     def store_recordmap(self, recordmap):
         for table, records in recordmap.items():
+            if records is None: continue
```

Comment:
There appears to be a field (`__json__`) in the user content that is occasionally set to `None`.  This simply does a quick check and skips adding any empty record sets to the store.

Reply:
Thank you for your work.  But still, I'm uneasy as to why this is occurring right now or this is the prelude to major API changes.

Reply:
Ha! This broke everything all at once for us 💯

Reply:
Are you referring to the change in this pull request or the bug?

Reply:
> Are you referring to the change in this pull request or the bug?

The bug! Thank you for the fix :)

Reply:
Sure thing...  It was a quick change, just making sure it didn't have some other consequences 😄

Reply:
For folks wanting to monkey patch their code while waiting on the merge:
`import notion
def store_recordmap(self, recordmap):
    for table, records in recordmap.items():
        if records is None: continue
        for id, record in records.items():
            self._update_record(
                table, id, value=record.get("value"), role=record.get("role")
            )
notion.store.RecordStore.store_recordmap = store_recordmap`

Reply:
I had to modify the first line in the above snippet to be `import notion.store` instead of `import notion` for it to work for me with python3.6 and notion==0.0.27. 

thank you for the workaround until this gets merged!